src/plots/geo/geo.js

```diff
@@ -647,9 +647,9 @@ function getProjection(geoLayout) {
     var projFn = geo[projName] || geoProjection[projName];
     var projection = projFn();
 
-    var clipAngle = geoLayout._isClipped ?
-        constants.lonaxisSpan[projType] / 2 :
-        null;
+    var clipAngle =
+        projType === 'satellite' ? Math.acos(1 / projection.distance()) * 180 / Math.PI :
```

Comment:
Where does `projection.distance()` come from? Is that linked to the `projection.scale` attribute? I imagine users of the satellite would want to be able to fix the satellite's location (height and center lat/lon) and still zoom and pan - to match up with a specific image taken by a satellite. Is that possible as it stands?

Reply:
The `distance` and `tilt` options are now implemented for `satellite` by https://github.com/plotly/plotly.js/pull/5801/commits/d102a470aac72bc0dd0f11a4c9295e09f76cff96.